package.json

```diff
@@ -16,6 +16,7 @@
     "clean:node_modules": "rm -rf node_modules",
     "clean": "npm run clean:dist && npm run clean:lerna && npm run clean:node_modules",
     "lint:scss": "stylelint 'packages/**/*.scss' --syntax scss",
+    "lint:scss:fix": "stylelint 'packages/**/*.scss' --syntax scss --fix",
```

Comment:
I think it makes sense to add this script and automatically fix scss on pre-commit (like it's done for js files)